tests/pyfunc/test_spark.py

```diff
@@ -1551,9 +1564,15 @@ def predict(self, context, model_input):
     assert pdf["output"][1] == ("b", [1], (False, ("another_string",)), [(0.2,), (0.3,)])
 
 
+@pytest.mark.parametrize("force_stdin_scoring_server", [False, True])
```

Comment:
why do we need to run this test with `MLFLOW_ENFORCE_STDIN_SCORING_SERVER_FOR_SPARK_UDF`? Does `MLFLOW_ENFORCE_STDIN_SCORING_SERVER_FOR_SPARK_UDF` affect how mlflow handles an invalid python path?

Reply:
make sense, I will remove this.